modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/image_selector/js/ImageSelector.js

```diff
@@ -51,16 +53,22 @@ const ImageSelector = ({
 }) => {
 	const [errorMessage, setErrorMessage] = useState('');
 	const [fileName, setFileName] = useState('');
-	const [image, setImage] = useState({
-		fileEntryId,
-		src: imageURL,
-	});
 	const [imageCropRegion, setImageCropRegion] = useState(
 		initialImageCropRegion
 	);
 	const [progressData, setProgressData] = useState();
 	const [progressValue, setProgressValue] = useState(0);
 
+	const [image, setImage] = useState({fileEntryId, src: imageURL});
+
+	useEffect(() => {
```

Comment:
Not sure if we have some rule/guideline about this, but I'd move this after all the methods definition, just next to the other `useEffect` to keep consistency.